iree/compiler/Codegen/Utils/Utils.h

```diff
@@ -57,6 +57,47 @@ ArrayRef<int64_t> getUntiledShape(Value tiledView);
 ArrayRef<int64_t> getUntiledResultShape(linalg::LinalgOp linalgOp,
                                         unsigned resultNum);
 
+/// Information about a tiled and distributed loop.
+///
+/// Right now distribution is happening as the same time when we tile the linalg
+/// op. 0) Given an original loop:
+///
+/// ```
+/// scf.for %iv = %init_lb to %init_ub step %init_step { ... }
+/// ```
+//
+/// 1) After tiling with tile size `%tile_size`, we have:
+//
+/// ```
+/// %tiled_step = %init_step * %tile_size
+/// scf.for %iv = %init_lb to %init_ub step %tiled_step { ... }
```

Comment:
nit: replace `tiled_step with `tile_size` for consistency?

Reply:
It's intentional so that it's a good contrast among `init_step`, `tiled_step`, and `dist_step`.